alembic_migration/versions/8bd0478eea2a_add_index_on_outing_date_end.py

```diff
@@ -0,0 +1,23 @@
+"""Add index on outing date_end
+
+Revision ID: 8bd0478eea2a
+Revises: 91b1beed9a1c
+Create Date: 2017-05-08 19:43:33.817581
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '8bd0478eea2a'
+down_revision = '91b1beed9a1c'
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    op.create_index(
+        'outing_date_end_idx', 'outings', ['date_end'], schema='guidebook')
```

Comment:
I would say I should be more something like:
```
Index(
            'ix_guidebook_òutings_date_end_and_document_id',
            date_end.desc(), document_id, postgresql_using='btree'),
```

But I don't have a database with enough data at hand to test.

Also, don't you also want to add the index to the model file?